Update diagnostic docs for SYSLIB0009 (#37452)
dotnet/docs: docs/fundamentals/syslib-diagnostics/syslib0009.md

ms.date: 10/20/2020
---

> # SYSLIB0009: AuthenticationManager is not supported. Methods will no-op or throw PlatformNotSupportedException

Reviewer: ```suggestion
# SYSLIB0009: AuthenticationManager is not supported
```

The following APIs are marked obsolete, starting in .NET 5. Use of these APIs generates warning `SYSLIB0009` at compile time and throws a <xref:System.PlatformNotSupportedException> at run time.

- <xref:System.Net.AuthenticationManager.Authenticate%2A?displayProperty=nameWithType>
- <xref:System.Net.AuthenticationManager.PreAuthenticate%2A?displayProperty=nameWithType>

> The entire <xref:System.Net.AuthenticationManager> class is marked as obsolete as of .NET 9. The usage of this class generates warning `SYSLIB0009` at compile time.

Reviewer: ```suggestion
In .NET 9 and later versions, the entire <xref:System.Net.AuthenticationManager> class is marked obsolete. Use of this class generates warning `SYSLIB0009` at compile time.
```

> The methods will no-op or throw <xref:System.PlatformNotSupportedException>.

Reviewer: ```suggestion
The methods in this class either no-op or throw a <xref:System.PlatformNotSupportedException> at run time.
```
